packages/snaps-execution-environments/src/proxy/ProxySnapExecutor.test.browser.ts

```diff
@@ -53,7 +54,7 @@ async function getResponse(
   });
 }
 
-describe('ProxySnapExecutor', () => {
+describe.skip('ProxySnapExecutor', () => {
```

Comment:
Why are we skipping?

Reply:
It required some additional configuration to get working. Enabled it again now.